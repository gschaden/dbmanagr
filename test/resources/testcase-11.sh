db-navigator.py -s skiline@localhost/skiline/
