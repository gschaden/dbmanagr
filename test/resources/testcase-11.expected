Id	Title	Subtitle	Autocomplete	
access_log	access_log	Title: {0}.id	skiline@localhost/skiline/access_log/
address	address	Title: {0}.id	skiline@localhost/skiline/address/
album	album	Title: {0}.id	skiline@localhost/skiline/album/
anon_mail	anon_mail	Title: {0}.id	skiline@localhost/skiline/anon_mail/
api_key	api_key	Title: {0}.comment	skiline@localhost/skiline/api_key/
api_key_2_badge_definition	api_key_2_badge_definition	Title: {api_key_id_title} || ': ' || {badge_definition_id_title}	skiline@localhost/skiline/api_key_2_badge_definition/
api_key_2_competition	api_key_2_competition	Title: {api_key_id_title} || ': ' || {competition_id_title}	skiline@localhost/skiline/api_key_2_competition/
api_permission	api_permission	Title: {api_key_id_title} || ': ' || {api_service_method_id_title}	skiline@localhost/skiline/api_permission/
api_service_method	api_service_method	Title: {0}.service || '.' || {0}.method	skiline@localhost/skiline/api_service_method/
audit_log	audit_log	Title: {0}.id	skiline@localhost/skiline/audit_log/
authentication_token	authentication_token	Title: {0}.id	skiline@localhost/skiline/authentication_token/
axess_connection_params	axess_connection_params	Title: {0}.id	skiline@localhost/skiline/axess_connection_params/
axess_lift_2_section	axess_lift_2_section	Title: {0}.id	skiline@localhost/skiline/axess_lift_2_section/
badge	badge	Title: {0}.id	skiline@localhost/skiline/badge/
badge_definition	badge_definition	Title: {0}.key	skiline@localhost/skiline/badge_definition/
badge_rule	badge_rule	Title: {0}.id	skiline@localhost/skiline/badge_rule/
badge_rule_2_ski_resort	badge_rule_2_ski_resort	Title: {0}.id	skiline@localhost/skiline/badge_rule_2_ski_resort/
badge_rule_processor_info	badge_rule_processor_info	Title: {0}.id	skiline@localhost/skiline/badge_rule_processor_info/
challenge	challenge	Title: {0}.id	skiline@localhost/skiline/challenge/
challenge_invitation	challenge_invitation	Title: {0}.id	skiline@localhost/skiline/challenge_invitation/
challenge_log	challenge_log	Title: {0}.id	skiline@localhost/skiline/challenge_log/
challenge_penalty	challenge_penalty	Title: {0}.id	skiline@localhost/skiline/challenge_penalty/
challenge_purchase	challenge_purchase	Title: {0}.id	skiline@localhost/skiline/challenge_purchase/
challenge_ranking_filter	challenge_ranking_filter	Title: {0}.id	skiline@localhost/skiline/challenge_ranking_filter/
check_missing_ids	check_missing_ids	Title: {0}.id	skiline@localhost/skiline/check_missing_ids/
checkpoint	checkpoint	Title: {0}.id	skiline@localhost/skiline/checkpoint/
chip_number_to_external_id_mapping	chip_number_to_external_id_mapping	Title: {0}.id	skiline@localhost/skiline/chip_number_to_external_id_mapping/
clicks	clicks	Title: {0}.id	skiline@localhost/skiline/clicks/
clicks_campaign	clicks_campaign	Title: {0}.id	skiline@localhost/skiline/clicks_campaign/
client	client	Title: {0}.id	skiline@localhost/skiline/client/
competition	competition	Title: {short_name_title}	skiline@localhost/skiline/competition/
competition_backup	competition_backup	Title: {0}.id	skiline@localhost/skiline/competition_backup/
competition_definition	competition_definition	Title: {0}.id	skiline@localhost/skiline/competition_definition/
competition_gradation	competition_gradation	Title: {0}.id	skiline@localhost/skiline/competition_gradation/
competition_gradation_item	competition_gradation_item	Title: {0}.id	skiline@localhost/skiline/competition_gradation_item/
competition_instance	competition_instance	Title: {0}.id	skiline@localhost/skiline/competition_instance/
competition_instance_2_badge_rule	competition_instance_2_badge_rule	Title: {0}.id	skiline@localhost/skiline/competition_instance_2_badge_rule/
competition_instance_2_checkpoint	competition_instance_2_checkpoint	Title: {0}.id	skiline@localhost/skiline/competition_instance_2_checkpoint/
competition_instance_2_ski_resort	competition_instance_2_ski_resort	Title: {0}.id	skiline@localhost/skiline/competition_instance_2_ski_resort/
competition_instance_aggregation	competition_instance_aggregation	Title: {0}.id	skiline@localhost/skiline/competition_instance_aggregation/
competition_invitation	competition_invitation	Title: {0}.id	skiline@localhost/skiline/competition_invitation/
competition_registration_key	competition_registration_key	Title: {0}.id	skiline@localhost/skiline/competition_registration_key/
competition_reminder_trial	competition_reminder_trial	Title: {0}.id	skiline@localhost/skiline/competition_reminder_trial/
competition_result	competition_result	Title: {0}.id	skiline@localhost/skiline/competition_result/
competition_result_format	competition_result_format	Title: {0}.id	skiline@localhost/skiline/competition_result_format/
competition_result_item	competition_result_item	Title: {0}.id	skiline@localhost/skiline/competition_result_item/
competition_terms	competition_terms	Title: {0}.id	skiline@localhost/skiline/competition_terms/
competition_type	competition_type	Title: {0}.id	skiline@localhost/skiline/competition_type/
constants	constants	Title: {0}.id	skiline@localhost/skiline/constants/
contact_collect	contact_collect	Title: {0}.id	skiline@localhost/skiline/contact_collect/
contact_info	contact_info	Title: {0}.id	skiline@localhost/skiline/contact_info/
countries	countries	Title: {0}.id	skiline@localhost/skiline/countries/
cp_transit	cp_transit	Title: {0}.id	skiline@localhost/skiline/cp_transit/
cpoint	cpoint	Title: {0}.id	skiline@localhost/skiline/cpoint/
daz	daz	Title: {0}.id	skiline@localhost/skiline/daz/
decorator	decorator	Title: {0}.id	skiline@localhost/skiline/decorator/
deleted_skicard	deleted_skicard	Title: {0}.id	skiline@localhost/skiline/deleted_skicard/
deleted_skicard_per_day	deleted_skicard_per_day	Title: {0}.id	skiline@localhost/skiline/deleted_skicard_per_day/
donation	donation	Title: {0}.id	skiline@localhost/skiline/donation/
donee	donee	Title: {0}.id	skiline@localhost/skiline/donee/
email_blacklist	email_blacklist	Title: {0}.id	skiline@localhost/skiline/email_blacklist/
email_targeting	email_targeting	Title: {0}.id	skiline@localhost/skiline/email_targeting/
email_template	email_template	Title: {0}.id	skiline@localhost/skiline/email_template/
equipment	equipment	Title: {0}.id	skiline@localhost/skiline/equipment/
equipment_brand	equipment_brand	Title: {0}.id	skiline@localhost/skiline/equipment_brand/
error_log	error_log	Title: {0}.id	skiline@localhost/skiline/error_log/
failed_ticket_request	failed_ticket_request	Title: {0}.id	skiline@localhost/skiline/failed_ticket_request/
faq_category	faq_category	Title: {0}.id	skiline@localhost/skiline/faq_category/
faq_item	faq_item	Title: {0}.id	skiline@localhost/skiline/faq_item/
favorite	favorite	Title: {0}.id	skiline@localhost/skiline/favorite/
firstname_2_gender	firstname_2_gender	Title: {0}.id	skiline@localhost/skiline/firstname_2_gender/
fitness_profile	fitness_profile	Title: {0}.id	skiline@localhost/skiline/fitness_profile/
forum	forum	Title: {0}.id	skiline@localhost/skiline/forum/
forum_visit	forum_visit	Title: {0}.id	skiline@localhost/skiline/forum_visit/
friend_invitation	friend_invitation	Title: {0}.id	skiline@localhost/skiline/friend_invitation/
google_pist_view	google_pist_view	Title: {0}.id	skiline@localhost/skiline/google_pist_view/
gpm_allg	gpm_allg	Title: {0}.id	skiline@localhost/skiline/gpm_allg/
gpm_cache	gpm_cache	Title: {0}.id	skiline@localhost/skiline/gpm_cache/
gpm_data	gpm_data	Title: {0}.id	skiline@localhost/skiline/gpm_data/
group	group	Title: {0}.id	skiline@localhost/skiline/group/
group_2_competition	group_2_competition	Title: {0}.id	skiline@localhost/skiline/group_2_competition/
group_invitation	group_invitation	Title: {0}.id	skiline@localhost/skiline/group_invitation/
handicap_log	handicap_log	Title: {0}.id	skiline@localhost/skiline/handicap_log/
hc_kiosk_competition_altimeters	hc_kiosk_competition_altimeters	Title: {0}.id	skiline@localhost/skiline/hc_kiosk_competition_altimeters/
hc_kiosk_competition_movies	hc_kiosk_competition_movies	Title: {0}.id	skiline@localhost/skiline/hc_kiosk_competition_movies/
hc_kiosk_competition_users	hc_kiosk_competition_users	Title: {0}.id	skiline@localhost/skiline/hc_kiosk_competition_users/
hc_kiosk_competitions	hc_kiosk_competitions	Title: {0}.id	skiline@localhost/skiline/hc_kiosk_competitions/
hc_schema_migrations	hc_schema_migrations	Title: {0}.id	skiline@localhost/skiline/hc_schema_migrations/
hc_user_to_kiosk_competitions	hc_user_to_kiosk_competitions	Title: {0}.id	skiline@localhost/skiline/hc_user_to_kiosk_competitions/
key_value_pair	key_value_pair	Title: {0}.id	skiline@localhost/skiline/key_value_pair/
kiosk	kiosk	Title: {0}.id	skiline@localhost/skiline/kiosk/
kiosk_competition_entry	kiosk_competition_entry	Title: {0}.id	skiline@localhost/skiline/kiosk_competition_entry/
kiosk_user_request	kiosk_user_request	Title: {0}.id	skiline@localhost/skiline/kiosk_user_request/
lang	lang	Title: {0}.id	skiline@localhost/skiline/lang/
layout	layout	Title: {0}.id	skiline@localhost/skiline/layout/
lead	lead	Title: {0}.id	skiline@localhost/skiline/lead/
lead_bmw_2010	lead_bmw_2010	Title: {0}.id	skiline@localhost/skiline/lead_bmw_2010/
lead_bmw_consent	lead_bmw_consent	Title: {0}.id	skiline@localhost/skiline/lead_bmw_consent/
lead_bmw_folder	lead_bmw_folder	Title: {0}.id	skiline@localhost/skiline/lead_bmw_folder/
lead_bmw_vehicle	lead_bmw_vehicle	Title: {0}.id	skiline@localhost/skiline/lead_bmw_vehicle/
localized_text	localized_text	Title: coalesce({0}.de, {0}.en, {0}.it)	skiline@localhost/skiline/localized_text/
location	location	Title: {0}.id	skiline@localhost/skiline/location/
login_history	login_history	Title: {0}.id	skiline@localhost/skiline/login_history/
mail_box	mail_box	Title: {0}.id	skiline@localhost/skiline/mail_box/
mail_box_item	mail_box_item	Title: {0}.id	skiline@localhost/skiline/mail_box_item/
mail_box_item_2_receiver	mail_box_item_2_receiver	Title: {0}.id	skiline@localhost/skiline/mail_box_item_2_receiver/
marketplace_sport2000rent	marketplace_sport2000rent	Title: {0}.id	skiline@localhost/skiline/marketplace_sport2000rent/
media	media	Title: {0}.id	skiline@localhost/skiline/media/
media_visit	media_visit	Title: {0}.id	skiline@localhost/skiline/media_visit/
messages	messages	Title: {0}.id	skiline@localhost/skiline/messages/
newsfeed	newsfeed	Title: {0}.id	skiline@localhost/skiline/newsfeed/
newsfeed_item	newsfeed_item	Title: {0}.id	skiline@localhost/skiline/newsfeed_item/
newsletter	newsletter	Title: {0}.id	skiline@localhost/skiline/newsletter/
newsletter_2_user	newsletter_2_user	Title: {0}.id	skiline@localhost/skiline/newsletter_2_user/
newsticker_item	newsticker_item	Title: {0}.id	skiline@localhost/skiline/newsticker_item/
photo	photo	Title: {0}.id	skiline@localhost/skiline/photo/
photo_point	photo_point	Title: {0}.id	skiline@localhost/skiline/photo_point/
photobook	photobook	Title: {0}.id	skiline@localhost/skiline/photobook/
poi	poi	Title: {0}.id	skiline@localhost/skiline/poi/
post	post	Title: {0}.id	skiline@localhost/skiline/post/
postal_code	postal_code	Title: {0}.id	skiline@localhost/skiline/postal_code/
province	province	Title: {0}.id	skiline@localhost/skiline/province/
qrtz_blob_triggers	qrtz_blob_triggers	Title: {0}.id	skiline@localhost/skiline/qrtz_blob_triggers/
qrtz_calendars	qrtz_calendars	Title: {0}.id	skiline@localhost/skiline/qrtz_calendars/
qrtz_cron_triggers	qrtz_cron_triggers	Title: {0}.id	skiline@localhost/skiline/qrtz_cron_triggers/
qrtz_fired_triggers	qrtz_fired_triggers	Title: {0}.id	skiline@localhost/skiline/qrtz_fired_triggers/
qrtz_job_details	qrtz_job_details	Title: {0}.id	skiline@localhost/skiline/qrtz_job_details/
qrtz_job_listeners	qrtz_job_listeners	Title: {0}.id	skiline@localhost/skiline/qrtz_job_listeners/
qrtz_locks	qrtz_locks	Title: {0}.id	skiline@localhost/skiline/qrtz_locks/
qrtz_paused_trigger_grps	qrtz_paused_trigger_grps	Title: {0}.id	skiline@localhost/skiline/qrtz_paused_trigger_grps/
qrtz_scheduler_state	qrtz_scheduler_state	Title: {0}.id	skiline@localhost/skiline/qrtz_scheduler_state/
qrtz_simple_triggers	qrtz_simple_triggers	Title: {0}.id	skiline@localhost/skiline/qrtz_simple_triggers/
qrtz_trigger_listeners	qrtz_trigger_listeners	Title: {0}.id	skiline@localhost/skiline/qrtz_trigger_listeners/
qrtz_triggers	qrtz_triggers	Title: {0}.id	skiline@localhost/skiline/qrtz_triggers/
race_profile	race_profile	Title: {0}.id	skiline@localhost/skiline/race_profile/
race_profile_equipment	race_profile_equipment	Title: {0}.id	skiline@localhost/skiline/race_profile_equipment/
rating	rating	Title: {0}.id	skiline@localhost/skiline/rating/
rejected_passwords	rejected_passwords	Title: {0}.id	skiline@localhost/skiline/rejected_passwords/
rel_daz_resort	rel_daz_resort	Title: {0}.id	skiline@localhost/skiline/rel_daz_resort/
rel_feratel_resort	rel_feratel_resort	Title: {0}.id	skiline@localhost/skiline/rel_feratel_resort/
rel_ggr_section	rel_ggr_section	Title: {0}.id	skiline@localhost/skiline/rel_ggr_section/
rel_ggr_section_exit	rel_ggr_section_exit	Title: {0}.id	skiline@localhost/skiline/rel_ggr_section_exit/
rel_pool_company	rel_pool_company	Title: {0}.id	skiline@localhost/skiline/rel_pool_company/
rel_resort_skicard_type	rel_resort_skicard_type	Title: {0}.id	skiline@localhost/skiline/rel_resort_skicard_type/
rel_resort_skidataconnection	rel_resort_skidataconnection	Title: {0}.id	skiline@localhost/skiline/rel_resort_skidataconnection/
rel_section_miz	rel_section_miz	Title: {0}.id	skiline@localhost/skiline/rel_section_miz/
rel_section_resort	rel_section_resort	Title: {0}.id	skiline@localhost/skiline/rel_section_resort/
rel_skicard_route	rel_skicard_route	Title: {0}.id	skiline@localhost/skiline/rel_skicard_route/
rel_skicardpd_route	rel_skicardpd_route	Title: {0}.id	skiline@localhost/skiline/rel_skicardpd_route/
rel_skicardpd_skicardsectionsupp	rel_skicardpd_skicardsectionsupp	Title: {0}.id	skiline@localhost/skiline/rel_skicardpd_skicardsectionsupp/
resort	resort	Title: trim({0}.name)	skiline@localhost/skiline/resort/
resort_connection_status	resort_connection_status	Title: {0}.id	skiline@localhost/skiline/resort_connection_status/
resort_details	resort_details	Title: {0}.id	skiline@localhost/skiline/resort_details/
resort_panorama	resort_panorama	Title: {0}.id	skiline@localhost/skiline/resort_panorama/
resort_panorama_section	resort_panorama_section	Title: {0}.id	skiline@localhost/skiline/resort_panorama_section/
resort_season_statistics	resort_season_statistics	Title: {0}.id	skiline@localhost/skiline/resort_season_statistics/
resort_sharing	resort_sharing	Title: {0}.id	skiline@localhost/skiline/resort_sharing/
resource_item	resource_item	Title: {0}.id	skiline@localhost/skiline/resource_item/
rfid_token	rfid_token	Title: {0}.id	skiline@localhost/skiline/rfid_token/
route	route	Title: {0}.id	skiline@localhost/skiline/route/
route_sections	route_sections	Title: {0}.id	skiline@localhost/skiline/route_sections/
routing_rules	routing_rules	Title: {0}.id	skiline@localhost/skiline/routing_rules/
routing_rules_last_section	routing_rules_last_section	Title: {0}.id	skiline@localhost/skiline/routing_rules_last_section/
schema_version	schema_version	Title: {0}.id	skiline@localhost/skiline/schema_version/
season	season	Title: {0}.id	skiline@localhost/skiline/season/
section	section	Title: {0}.id	skiline@localhost/skiline/section/
section_2_webcam	section_2_webcam	Title: {0}.id	skiline@localhost/skiline/section_2_webcam/
section_capacity	section_capacity	Title: {0}.id	skiline@localhost/skiline/section_capacity/
section_queuetime	section_queuetime	Title: {0}.id	skiline@localhost/skiline/section_queuetime/
section_type	section_type	Title: {0}.id	skiline@localhost/skiline/section_type/
security_info	security_info	Title: 'DOI: ' || coalesce({0}.doi, '-') || ', locked: ' || {0}.locked	skiline@localhost/skiline/security_info/
ski_diary	ski_diary	Title: {0}.id	skiline@localhost/skiline/ski_diary/
ski_handicap	ski_handicap	Title: {0}.id	skiline@localhost/skiline/ski_handicap/
ski_resort_info	ski_resort_info	Title: {0}.id	skiline@localhost/skiline/ski_resort_info/
ski_resort_performance	ski_resort_performance	Title: {0}.id	skiline@localhost/skiline/ski_resort_performance/
ski_resort_province	ski_resort_province	Title: {0}.id	skiline@localhost/skiline/ski_resort_province/
skicard	skicard	Title: {0}.external_id	skiline@localhost/skiline/skicard/
skicard_per_day	skicard_per_day	Title: {0}.id	skiline@localhost/skiline/skicard_per_day/
skicard_section_supplement	skicard_section_supplement	Title: {0}.id	skiline@localhost/skiline/skicard_section_supplement/
skidata_card_type_validity	skidata_card_type_validity	Title: {0}.id	skiline@localhost/skiline/skidata_card_type_validity/
skidata_card_types	skidata_card_types	Title: {0}.id	skiline@localhost/skiline/skidata_card_types/
skidata_connection_params	skidata_connection_params	Title: {0}.id	skiline@localhost/skiline/skidata_connection_params/
skidata_person_type	skidata_person_type	Title: {0}.id	skiline@localhost/skiline/skidata_person_type/
skidata_pool_2_resort	skidata_pool_2_resort	Title: {0}.id	skiline@localhost/skiline/skidata_pool_2_resort/
skidata_sections	skidata_sections	Title: {0}.id	skiline@localhost/skiline/skidata_sections/
skier_level	skier_level	Title: {0}.id	skiline@localhost/skiline/skier_level/
skiline_access_log	skiline_access_log	Title: {0}.id	skiline@localhost/skiline/skiline_access_log/
skiline_boot_log	skiline_boot_log	Title: {0}.id	skiline@localhost/skiline/skiline_boot_log/
skiline_error_log	skiline_error_log	Title: {0}.id	skiline@localhost/skiline/skiline_error_log/
skiline_statistics	skiline_statistics	Title: {0}.id	skiline@localhost/skiline/skiline_statistics/
skimovie_category	skimovie_category	Title: {0}.id	skiline@localhost/skiline/skimovie_category/
skimovie_statistics	skimovie_statistics	Title: {0}.id	skiline@localhost/skiline/skimovie_statistics/
skimovie_visit	skimovie_visit	Title: {0}.id	skiline@localhost/skiline/skimovie_visit/
spot	spot	Title: {0}.id	skiline@localhost/skiline/spot/
spot_2_user	spot_2_user	Title: {0}.id	skiline@localhost/skiline/spot_2_user/
svg_logo	svg_logo	Title: {0}.id	skiline@localhost/skiline/svg_logo/
svg_template	svg_template	Title: {0}.id	skiline@localhost/skiline/svg_template/
svg_template_2_ski_resort	svg_template_2_ski_resort	Title: {0}.id	skiline@localhost/skiline/svg_template_2_ski_resort/
svg_template_content	svg_template_content	Title: {0}.id	skiline@localhost/skiline/svg_template_content/
swisscom_performance_status	swisscom_performance_status	Title: {0}.id	skiline@localhost/skiline/swisscom_performance_status/
swisscom_skimovie_status	swisscom_skimovie_status	Title: {0}.id	skiline@localhost/skiline/swisscom_skimovie_status/
system_property	system_property	Title: {0}.value	skiline@localhost/skiline/system_property/
test_skicards	test_skicards	Title: {0}.id	skiline@localhost/skiline/test_skicards/
thread	thread	Title: {0}.id	skiline@localhost/skiline/thread/
thread_visit	thread_visit	Title: {0}.id	skiline@localhost/skiline/thread_visit/
top100_sharing	top100_sharing	Title: {0}.id	skiline@localhost/skiline/top100_sharing/
track	track	Title: {0}.id	skiline@localhost/skiline/track/
usage_cache	usage_cache	Title: {0}.id	skiline@localhost/skiline/usage_cache/
user	user	Title: {0}.fname || ' ' || {0}.lname	skiline@localhost/skiline/user/
user_2_checkpoint	user_2_checkpoint	Title: {0}.id	skiline@localhost/skiline/user_2_checkpoint/
user_2_competition	user_2_competition	Title: {0}.id	skiline@localhost/skiline/user_2_competition/
user_2_group	user_2_group	Title: {0}.id	skiline@localhost/skiline/user_2_group/
user_2_language	user_2_language	Title: {0}.id	skiline@localhost/skiline/user_2_language/
user_2_role	user_2_role	Title: {0}.id	skiline@localhost/skiline/user_2_role/
user_2_route	user_2_route	Title: {0}.id	skiline@localhost/skiline/user_2_route/
user_2_ski_resort	user_2_ski_resort	Title: {0}.id	skiline@localhost/skiline/user_2_ski_resort/
user_2_skimovie	user_2_skimovie	Title: {0}.id	skiline@localhost/skiline/user_2_skimovie/
user_2_user	user_2_user	Title: {0}.id	skiline@localhost/skiline/user_2_user/
user_2_welcome_email	user_2_welcome_email	Title: {0}.id	skiline@localhost/skiline/user_2_welcome_email/
user_agent	user_agent	Title: {0}.id	skiline@localhost/skiline/user_agent/
user_alarm	user_alarm	Title: {0}.id	skiline@localhost/skiline/user_alarm/
user_competition_reminder_email	user_competition_reminder_email	Title: {0}.id	skiline@localhost/skiline/user_competition_reminder_email/
user_facebook_info	user_facebook_info	Title: {0}.id	skiline@localhost/skiline/user_facebook_info/
user_feature	user_feature	Title: {0}.id	skiline@localhost/skiline/user_feature/
user_season_statistics	user_season_statistics	Title: {0}.id	skiline@localhost/skiline/user_season_statistics/
user_visit	user_visit	Title: {0}.id	skiline@localhost/skiline/user_visit/
video	video	Title: {0}.id	skiline@localhost/skiline/video/
video_annotations	video_annotations	Title: {0}.id	skiline@localhost/skiline/video_annotations/
video_details	video_details	Title: {0}.id	skiline@localhost/skiline/video_details/
webcams	webcams	Title: {0}.id	skiline@localhost/skiline/webcams/

